x/oracle/abci.go

```diff
@@ -10,7 +10,7 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 )
 
-// IsPeriodLastBlock returns true if we are at the last block of the period
+// BADIsPeriodLastBlock returns true if we are at the last block of the period
```

Comment:
Where are these "BAD", "XXX" comment pre/suffixes coming from? I don't see them in main

Reply:
No idea :D, let me see in commits.

Reply:
I added it to test the linter.

Reply:
removing